error:060E90C0:digital envelope routines:aesni_xts_init_key:xts duplicated keys

The remaining error after https://github.com/pyca/cryptography/issues/4884 with Fedora's openssl 1.1.1b-10:

```
$ docker run -ti fedora:rawhide /bin/bash

# dnf install https://kojipkgs.fedoraproject.org//packages/openssl/1.1.1b/10.fc31/x86_64/openssl{,-{devel,libs}}-1.1.1b-10.fc31.x86_64.rpm
...
Installed:
  openssl-devel-1:1.1.1b-10.fc31.x86_64

# dnf install /usr/bin/git tox gcc
# git clone https://github.com/pyca/cryptography.git
# cd cryptography
# tox
...
_________________ test_buffer_protocol_alternate_modes[mode5] __________________

mode = <cryptography.hazmat.primitives.ciphers.modes.XTS object at 0x7ff4c8e56c88>
backend = <cryptography.hazmat.backends.openssl.backend.Backend object at 0x7ff4d28343c8>

    @pytest.mark.parametrize(
        "mode",
        [
            modes.CBC(bytearray(b"\x00" * 16)),
            modes.CTR(bytearray(b"\x00" * 16)),
            modes.OFB(bytearray(b"\x00" * 16)),
            modes.CFB(bytearray(b"\x00" * 16)),
            modes.CFB8(bytearray(b"\x00" * 16)),
            modes.XTS(bytearray(b"\x00" * 16)),
        ]
    )
    @pytest.mark.requires_backend_interface(interface=CipherBackend)
    def test_buffer_protocol_alternate_modes(mode, backend):
        data = bytearray(b"sixteen_byte_msg")
        cipher = base.Cipher(
            algorithms.AES(bytearray(b"\x00" * 32)), mode, backend
        )
>       enc = cipher.encryptor()

tests/hazmat/primitives/test_aes.py:495: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
.tox/py37/lib/python3.7/site-packages/cryptography/hazmat/primitives/ciphers/base.py:121: in encryptor
    self.algorithm, self.mode
.tox/py37/lib/python3.7/site-packages/cryptography/hazmat/backends/openssl/backend.py:298: in create_symmetric_encryption_ctx
    return _CipherContext(self, cipher, mode, _CipherContext._ENCRYPT)
.tox/py37/lib/python3.7/site-packages/cryptography/hazmat/backends/openssl/ciphers.py:116: in __init__
    self._backend.openssl_assert(res != 0)
.tox/py37/lib/python3.7/site-packages/cryptography/hazmat/backends/openssl/backend.py:128: in openssl_assert
    return binding._openssl_assert(self._lib, ok)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

lib = <module 'lib'>, ok = False

    def _openssl_assert(lib, ok):
        if not ok:
            errors = _consume_errors(lib)
            errors_with_text = []
            for err in errors:
                buf = ffi.new("char[]", 256)
                lib.ERR_error_string_n(err.code, buf, len(buf))
                err_text_reason = ffi.string(buf)
    
                errors_with_text.append(
                    _OpenSSLErrorWithText(
                        err.code, err.lib, err.func, err.reason, err_text_reason
                    )
                )
    
            raise InternalError(
                "Unknown OpenSSL error. This error is commonly encountered when "
                "another library is not cleaning up the OpenSSL error stack. If "
                "you are using cryptography with another library that uses "
                "OpenSSL try disabling it before reporting a bug. Otherwise "
                "please file an issue at https://github.com/pyca/cryptography/"
                "issues with information on how to reproduce "
                "this. ({0!r})".format(errors_with_text),
>               errors_with_text
            )
E           cryptography.exceptions.InternalError: Unknown OpenSSL error. This error is commonly encountered when another library is not cleaning up the OpenSSL error stack. If you are using cryptography with another library that uses OpenSSL try disabling it before reporting a bug. Otherwise please file an issue at https://github.com/pyca/cryptography/issues with information on how to reproduce this. ([_OpenSSLErrorWithText(code=101617856, lib=6, func=233, reason=192, reason_text=b'error:060E90C0:digital envelope routines:aesni_xts_init_key:xts duplicated keys')])

.tox/py37/lib/python3.7/site-packages/cryptography/hazmat/bindings/openssl/binding.py:78: InternalError
...
```

Full tox output for `py37`: [py37.log](https://github.com/pyca/cryptography/files/3167434/py37.log)

--- a/tests/hazmat/primitives/test_aes.py
+++ b/tests/hazmat/primitives/test_aes.py
@@ -490,7 +490,7 @@ class TestAESModeGCM(object):
 def test_buffer_protocol_alternate_modes(mode, backend):
     data = bytearray(b"sixteen_byte_msg")
     cipher = base.Cipher(
-        algorithms.AES(bytearray(b"\x00" * 32)), mode, backend
+        algorithms.AES(bytearray(os.urandom(32))), mode, backend
     )
     enc = cipher.encryptor()
     ct = enc.update(data) + enc.finalize()

